### PR TITLE
add support for custom tag in gitops template

### PR DESCRIPTION
**Pull request by askvinni**
### Type of Change

- [ ] Bugfix
- [X] Enhancement / new feature
- [ ] Refactoring
- [ ] Documentation

### Description

see https://github.com/Staffbase/gitops-github-action/pull/89

### Checklist

- [x] Add relevant labels (for example type of change or patch/minor/major)
- [ ] Make sure not to introduce some mistakes
- [X] Update documentation
- [ ] Review the [Contributing Guideline](../blob/main/CONTRIBUTING.md) and sign CLA
- [ ] Reference relevant issue(s) and close them after merging

### CODE REVIEW

**Comment by askvinni** [comment]
I have read the CLA Document and I hereby sign the CLA

**Comment by askvinni** [comment]
recheck

**Review by flaxel — COMMENTED** [review]
If I understand the change correctly, the custom tag is always preferred. Does this mean that you only create releases with the custom tag? Otherwise, you use the normal logic? 🤔 If this is the case, I would adapt the documentation as it only affects the prod tag.

**Comment by askvinni** [comment]
> If I understand the change correctly, the custom tag is always preferred. Does this mean that you only create releases with the custom tag? Otherwise, you use the normal logic? 🤔 If this is the case, I would adapt the documentation as it only affects the prod tag.

Correct; we have a monorepo and want to tag with `prefix/version`, and use the action to release code within the `prefix` folder with version `version`, so an initial step extracts both variables and the gitops workflow uses them.

I'll update the docs and rerequest review.

File: README.md [hunk 1]
```diff
@@ -133,6 +133,8 @@ jobs:
       docker-file: <path-to-Dockerfile>
       # optional: name of the docker image, default: private/<repository_name>
       docker-image: <your-image>
+      # optional: custom tag for the docker image. If set, will always be preferred from the tag generated by the workflow.
```
Comment (flaxel) [inline README.md:136]:
```suggestion
      # optional: custom tag for the productive docker image which is preferred over the tag generated by the workflow
```

**Review by flaxel — APPROVED** [review]

**Pull request merged**